src/rstxml2db/rstxml2db.xsl

```diff
@@ -705,8 +705,60 @@
     </informalfigure>
   </xsl:template>
 
-  <xsl:template match="paragraph[strong][preceding-sibling::figure]"/>
+	<xsl:template match="paragraph[strong][preceding-sibling::figure]"/>
+
+	<xsl:template match="desc">
+    <xsl:variable name="id" select="normalize-space(desc_signature/@ids)"/>
+	  <variablelist>
+      <xsl:message>INFO: ID "<xsl:value-of select="$id"/>"</xsl:message>
```

Comment:
Maybe we can disable the `xsl:message` here`? I guess, we don't need it anymore, right?